### ISSUE TITLE
2.10.11 run noticeably slower

### ISSUE THREAD

**Issue by Mrcubix** [nekomangaorg/Neko#1138]
### Steps to reproduce

1. Use 20.10.10
2. Scrolling is for the most part smooth and the app doesn't freeze every couple seconds, even when reading, interface is responsive, ect...
3. Update to 20.10.11
4. App is slower, freezes when scrolling, buttons take up to seconds to actually tap

### Expected behavior

Performance should be the same or better as 20.10.10

### Actual behavior

- App constantly stutter while scrolling my library & reading a manga, no matter if it has long strips or not.
- Buttons like settings take seconds to actually react,
- When it does, it take more for the settings view to actually show on screen
- Same issues apply for pretty much the entirety of the app

### Neko version

20.10.11

### Android version

Android 7.1.2, Android 8.0.0

### Device

SM-T560, SM-J600F

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by Mrcubix** [comment]
If neccessary, i can try debugging the app provided someone provide steps, else i will just downgrade until fixed

**Comment by CarlosEsco** [comment]
Nothing of note was changed in the screens you mentioned between 2.10.10 and 2.10.11.

**Comment by Mrcubix** [comment]
If by the screens i mentioned you mean any screens then there is a bigger issue.

My main guess for the library and explore screens would be the way thumbnails are displayed

because i know a small few of them weren't loading before and suddenly neko load them and performance drop significantly

As for the other screens, i haven't noticed any visible changes other than performance being way worse, tho that may be dude to the former.

**Comment by CarlosEsco** [comment]
What i am saying is things you are saying are having performance issues, were not changed.  If you can upload a video of the issues that could be helpful.

**Comment by Mrcubix** [comment]
sure lemme get scrcpy and a clean reboot goin

**Comment by Mrcubix** [comment]
https://user-images.githubusercontent.com/39861216/191056143-dbbc9990-b4ca-4f19-b7a3-4c20c091423c.mp4

sure enough, 30 seconds after the video was recorded, neko now doesn't respond at all and i'm forced to close the app

![image](https://user-images.githubusercontent.com/39861216/191056117-7fd9b0c5-4942-4ec4-9c24-2ef333aca7a2.png)

**Comment by CarlosEsco** [comment]
Can you backup your library, downgrade to 2.10.10 restore. then take a video

**Comment by Mrcubix** [comment]
waiting on whichever thumbnails want to load first to do before i start recording for fair comparison, which take a bit of time

**Comment by Mrcubix** [comment]
https://user-images.githubusercontent.com/39861216/191079553-2cd0e7fe-a1f1-4f4f-ad68-d78f8fdd84bd.mp4

Scrolling while reading is slightly smoother in 2.10.10 >
Scrolling at fast speed in library is slower in 2.10.10 >
Scrolling at medium speed in library is about the same =
Scrolling at slower speed in library is faster on 2.10.10 <
Scrolling in a category with a few mangas is way smoother in 2.10.10 >>

Settings Navigation is smoother and buttons react slightly faster in 2.10.10 >

Switching categories is the same
Startup is the same

**Comment by nonproto** [comment]
lot has changed since this was opened
